ProteoformSuiteInternal/ProteoformCommunity.cs

```diff
@@ -141,22 +140,11 @@ private static List<ProteoformRelation> count_nearby_relations(List<ProteoformRe
 
         public Dictionary<string, List<ProteoformRelation>> relate_ed()
         {
-            //TODO: simply fy with ?: operator
             Dictionary<string, List<ProteoformRelation>> ed_relations = new Dictionary<string, List<ProteoformRelation>>();
-            if (!Lollipop.limit_TD_BU_theoreticals)
-            {
-                Parallel.ForEach(decoy_proteoforms, decoys =>
-                {
-                    ed_relations[decoys.Key] = relate_et(experimental_proteoforms.Where(p => p.accepted).ToArray(), decoys.Value, ProteoformComparison.ed);
-                });
-            }
-            else
-            {
                 Parallel.ForEach(decoy_proteoforms, decoys =>
                 {
-                    ed_relations[decoys.Key] = relate_et(experimental_proteoforms.Where(p => p.accepted).ToArray(), decoys.Value.Where(t => t.psm_count_BU > 0).ToArray(), ProteoformComparison.ed);
+                    ed_relations[decoys.Key] = Lollipop.limit_TD_BU_theoreticals? relate_et(experimental_proteoforms.Where(p => p.accepted).ToArray(), decoys.Value.Where(t => t.psm_count_BU > 0).ToArray(), ProteoformComparison.ed) : relate_et(experimental_proteoforms.Where(p => p.accepted).ToArray(), decoys.Value, ProteoformComparison.ed);
```

Comment:
Ooh much cleaner. Could you put them on separate lines, so the expressions are evaluated separately by Coveralls? For example:
`````
something = booleanexpression ?
          another_thing :
          yet_another_thing;
`````